kinetic/distribution.yaml

```
@@ -15029,13 +15029,18 @@ repositories:
       tags:
         release: release/kinetic/{package}/{version}
       url: https://github.com/amilankovich-slab/swarmros-release.git
+      version: 0.3.1-4
     source:
       test_pull_requests: true
       type: git
       url: https://github.com/amilankovich-slab/swarmros.git
       version: master
     status: developed
     status_description: developed
+    doc:
+      type: git
+      url: https://github.com/amilankovich-slab/swarmros.git
+      version: 0.3.1-4
```

This version needs to be a branch (or tag) in the upstream repo. Usually it's set to the same branch as the source entry above so `master`.
```suggestion
      version: master
```